canonical-websites/cn.ubuntu.com
# Add hackathon takeover

## Done

Added a new homepage takeover

## QA

-`./run`
- Go to [the homepage](http://localhost:8010/)
- Check the takeover in all viewports
- Compare with [the design](https://app.zenhub.com/workspace/o/canonical-websites/cn.ubuntu.com/issues/145#issuecomment-322487771)

## Issue / Card

Fixes #145
